spec/service/food_service_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe FoodService do
+  describe '#class_methods' do
+    it 'returns foods that contain an ingredient searched by keyword' do
+      fs = FoodService.search('sweet potatoes')
+
+      expect(fs).to be_a(Hash)
+      expect(fs[:foods]).to be_a(Array)
+      expect(fs[:totalHits]).to eq(49676)
+      expect(fs[:foods].count).to eq(50)
```

Comment:
I think I would have limited the `:pageSize` (I think that is the endpoint to limit) to 10, since we only want to get 10 results.